.changes/v3.9.0/965-features.md

```diff
@@ -0,0 +1,2 @@
+* **New Resource:** `vcd_rde_interface` to manage the Defined Interfaces required to use RDE types [GH-965]
```

Comment:
Trying to suggest something self-explanatory for first-time users. Maybe like this?

```suggestion
* **New Resource:** `vcd_rde_interface` to manage Runtime Defined Entity Interfaces which are required for using Runtime Defined Entity (RDE) types [GH-965]
```

Reply:
Changed.